hantec_api_ecommerce/controllers/main.py

```diff
@@ -0,0 +1,570 @@
+
+from odoo.http import request, Response, Controller, route
+import json
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class MainController(Controller):
+    @route('/create_contact', methods=['POST'], type='json', auth='user' )
```

Comment:
```suggestion
    @route("/create_contact", methods=["POST"], type="json", auth="user")
```
Use double quoting please, check all coincidences